src/features/profile/l10n/messageIds.ts

```diff
@@ -36,4 +36,8 @@ export default makeMessages('feat.profile', {
     profile: m('Profile'),
     timeline: m('Timeline'),
   },
+  user: {
+    hasAccount: m('Has Zetkin account'),
+    noAccount: m('No Zetkin account'),
```

Comment:
Was there any conversation about this wording? It feels a bit off to me, but maybe that's just because I'm used to other keywords, and this is actually more understandable.

In Gen2, I think it says "Connected to Zetkin account". I don't know if that's better.

Maybe the small difference for me is in the keyword "connected" (rather than "has"), and when I try to analyze my own thinking, I believe that it's because I think of the person profile page as representing a physical person. That physical person can **have** a Zetkin account, without us knowing. It's only if that Zetkin account is **connected** to this particular profile that it will show up here.

But maybe that's too much of a technical distinction and doesn't really help with understanding for normal users. I trust your judgement in assessing that. What do you think?

Reply:
We tried a bunch of different potentials and i felt like let's just go for something and discuss it more when we can interact with it. 
I think the suggestion you propose makes sense and is a good wording - also if it's a phrasing people are used to it makes sense to keep it i think? I want to change it to your suggestion.

Reply:
![bild](https://github.com/zetkin/app.zetkin.org/assets/58265097/844acc43-9b8a-4ce6-bade-13b111249639)
changing to: 
"Connected to a Zetkin account" and "Not connected to a Zetkin account"